proguard/proguard-square-okhttp3.pro

```diff
@@ -0,0 +1,6 @@
+# OkHttp
+-keepattributes Signature
+-keepattributes *Annotation*
+-keep class okhttp3.** { *; }
+-keep interface okhttp3.** { *; }
+-dontwarn okhttp3.**
```

Comment:
it proguard rules are massively over matching.

i seems better official proguard rule https://github.com/square/okhttp#proguard